examples/server/bench/script.js

```diff
@@ -89,6 +90,9 @@ export default function () {
         ],
         "model": model,
         "stream": true,
+        "stream_options": {
+          "include_usage": true, // False to be supported in llama.cpp server
```

Comment:
Not sure what you mean here, but in llama.cpp we ignore `include_usage` and always include include the usage info.